### DISCUSSION TITLE
Why is yahoomail.com flagged as malware?

### DISCUSSION THREAD

**Discussion by d3vansh36** [T145/black-mirror#208] [Q&A] [answered]
Why is yahoomail.com flagged as malware? Even tho it redirects to the legit yahoo page, ie. mail.yahoo.com.

**Comment by T145** [comment, marked as answer]
The content in Black Mirror is not all "malware." My guess is a list provider caught unsavory content from a Yahoo email or detected that a Yahoo user was targeted by similar content. I'm happy to whitelist that domain as it presently redirects to `yahoo.com`.

**Reply by d3vansh36** [reply]
Yeah that works! Also, there are a lot of false positives in ips and domains. When I cross check any with virus total/abuseipdb it doesn't come out to be malicious.

**Reply by T145** [reply]
Again `Black Mirror` isn't completely about blocking malware. So in the context of this project, a "false positive" would be any host that doesn't fall within the guidelines specified in [the manifesto](https://github.com/T145/black-mirror/blob/master/MANIFESTO.md). Why is there a manifesto or guidelines at all? To help people know what to expect from the list content. I've used countless blocklists without reviewing their content and trusting what they advertised only to discover through future review they had many "wild edits," or blocks that didn't adhere to whatever was advertised. I'd have to spend a long time manually whitelisting, which is largely why this project came into existence.
